assembly/src/parsers/nodes.rs

```diff
@@ -269,6 +270,7 @@ impl fmt::Display for Instruction {
             Self::DivImm(value) => write!(f, "div.{value}"),
             Self::Neg => write!(f, "neg"),
             Self::Inv => write!(f, "inv"),
+            Self::Incr => write!(f, "incr"),
```

Comment:
We don't have `incr` instruction in the assembly - so, this should be printed out as `add.1`.